<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportFDSNWS</class>
 <widget class="QDialog" name="ImportFDSNWS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FDSN web service - Stations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QScrollArea" name="sideBar">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>90</width>
          <height>948</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableStations">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Network</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Station</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Location</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Channel</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>SPS</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Lat</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Lon</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Elevation</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sensor</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnFetch">
       <property name="text">
        <string>Fetch</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableStations</tabstop>
  <tabstop>btnFetch</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportFDSNWS</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>1106</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>1092</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportFDSNWS</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>1109</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>1094</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
